src/VCAuthn/Utils/UrlUtils.cs

```diff
@@ -0,0 +1,15 @@
+namespace VCAuthn.Utils
+{
+    public static class UrlUtils
+    {
+        public static string EnsureLeadingSlash(this string url)
+        {
+            if (!url.StartsWith("/"))
+            {
+                return "/" + url;
+            }
```

Comment:
I feel like we should invert this for readability? What do you think?

```csharp
if (url.StartsWith("/"))
{
    return url;
}
return $"/{url}";
```

Reply:
yeah, I like yours more 👍